codegen-sbt/src/sbt-test/compiletime-codegen/simple/modules/server/src/main/scala/poc/generator/CalibanClientGenerator.scala

```diff
@@ -0,0 +1,10 @@
+package poc.generator
+
+import caliban.tools.compiletime.CompileTime
+import io.guizmaii.poc.caliban.server.GraphQLApi
+import zio.{ExitCode, URIO}
+
+private[generator] object CalibanClientGenerator extends zio.App {
+  override def run(args: List[String]): URIO[zio.ZEnv, ExitCode] =
+    CompileTime.generateClient(args, GraphQLApi.api)
```

Comment:
Since the user needs to write that code, how about passing all arguments by Scala code? It's actually much simpler to use than doing it from sbt.

Reply:
Hum. I didn't think about this possibility and yes, it's way simpler. Let me check 🤔

Reply:
Done :)